CareKit/Connect/OCKConnectTableViewCell.m

```diff
@@ -102,7 +103,12 @@ - (void)updateView {
     } else {
         _imageView.image = nil;
         _imageView.backgroundColor = OCKSystemGrayColor();
-        _monogramLabel.text = self.contact.monogram;
+        
+        NSString *monogram = self.contact.monogram;
+        NSRange stringRange = {0, MIN([monogram length], 2)};
+        stringRange = [monogram rangeOfComposedCharacterSequencesForRange:stringRange];
+        NSString *clippedMonogram = [monogram substringWithRange:stringRange];
+        _monogramLabel.text = clippedMonogram;
```

Comment:
@umerkhan-apple Will do though I'll be gone for a few days so I will get to it when I get back. However while subStringToIndex is simpler it breaks surrogate pairs. Is the simplify worth it in this case? When I use subStringToIndex with my example above the emoji becomes a half 'A' in a box unicode character (sorry I don't know the exact name).